pkg/kv/dynamodb/store.go

```diff
@@ -55,18 +67,29 @@ func init() {
 // Open - opens and returns a KV store over DynamoDB. This function creates the DB session
 // and sets up the KV table. dsn is a string with the DynamoDB endpoint
 func (d *Driver) Open(ctx context.Context, dsn string) (kv.Store, error) {
+	// TODO: Get table name from env
+	params, err := parseDsn(dsn)
+	if err != nil {
+		return nil, fmt.Errorf("%w: %s", kv.ErrDriverConfiguration, err)
```

Comment:
When the caller gets this error back, what do you expect the error type to be? More precisely, what would you like to happen (I think you can only have one): `errors.Is(err, kv.ErrDriverConfiguration) == true` or `errors.Is(err, json.ParsingFailure) == true` as an example.